src/UnifiedAPIClient.ts

```diff
@@ -0,0 +1,171 @@
+import { AxiosRequestConfig } from 'axios';
+
+import { BaseRestClient } from './lib/BaseRestClient.js';
+import {
+  REST_CLIENT_TYPE_ENUM,
+  RestClientOptions,
+  RestClientType,
+} from './lib/requestUtils.js';
+import {
+  GetAnnouncementsRequestUTA,
+  GetCurrencyRequestUTA,
+  GetCurrentFundingRateRequestUTA,
+  GetHistoryFundingRateRequestUTA,
+  GetKlinesRequestUTA,
+  GetOrderBookRequestUTA,
+  GetServiceStatusRequestUTA,
+  GetSymbolRequestUTA,
+  GetTickerRequestUTA,
+  GetTradesRequestUTA,
+} from './types/request/uta-types.js';
+import { APISuccessResponse } from './types/response/shared.types.js';
+import {
+  GetAnnouncementsResponseUTA,
+  GetCrossMarginConfigResponseUTA,
+  GetCurrencyResponseUTA,
+  GetCurrentFundingRateResponseUTA,
+  GetHistoryFundingRateResponseUTA,
+  GetKlinesResponseUTA,
+  GetOrderBookResponseUTA,
+  GetServiceStatusResponseUTA,
+  GetSymbolResponseUTA,
+  GetTickerResponseUTA,
+  GetTradesResponseUTA,
+} from './types/response/uta-types.js';
+
+/**
+ * Unified Trading Account Client
+ *
+ * This client provides access to the Unified Trading Account API endpoints
+ * that unify market data access across Spot, Futures, and Margin trading.
+ */
+export class UnifiedAPIClient extends BaseRestClient {
+  constructor(
+    restClientOptions: RestClientOptions = {},
+    requestOptions: AxiosRequestConfig = {},
+  ) {
+    super(restClientOptions, requestOptions);
+    return this;
+  }
+
+  getClientType(): RestClientType {
+    return REST_CLIENT_TYPE_ENUM.unifiedTradingAccount;
+  }
+
+  /**
+   *
+   * REST - Unified Trading Account - Market Data
+   *
+   */
+
+  /**
+   * Get Announcements
+   * This interface can obtain the latest news announcements, and the default
+   * page search is for announcements within a month.
+   */
+  getAnnouncements(
+    params?: GetAnnouncementsRequestUTA,
+  ): Promise<APISuccessResponse<GetAnnouncementsResponseUTA>> {
+    return this.get('api/ua/v1/market/announcement', params);
+  }
+
+  /**
+   * Get Currency
+   * Request the currency details of a specified currency via this endpoint.
+   */
+  getCurrency(
+    params?: GetCurrencyRequestUTA,
+  ): Promise<APISuccessResponse<GetCurrencyResponseUTA>> {
+    return this.get('api/ua/v1/market/currency', params);
+  }
+
+  /**
+   * Get Symbol
+   * Request a list of available currency pairs for trading via this endpoint.
+   */
+  getSymbol(
```

Comment:
```suggestion
  getSymbols(
```

Worth setting expectations in the phrasing that this is a plural response.

Also unsure whether to embed the response type in a more readable way, so it's obvious there's a nested list - thoughts?
```
): Promise<APISuccessResponse<{
  tradeType: string;
  list: SymbolUTA[];
}>> {
```